Update sf-rest-connector-example (#1449)
wso2/docs-ei: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

            - **refreshToken** : Value of the refresh token.
       
            <img src="../../../../assets/img/connectors/salesforce-api-init-operation-sequnce.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
           
    2. Setting up the `create` operation.

>         1. Then we will setup the `create` operation configurations. In this operation we are going to create a `sObjects` in the Salesforce account. An `SObject` represents a specific table in the database that you can discretely query. It describes the individual metadata for the specified object. Please find the `create` operation parameters listed here.

Reviewer: ```suggestion
        1. Setup the `create` operation configurations. In this operation we are going to create `sObjects` in the Salesforce account. An `sObject` represents a specific table in the database that you can discretely query. It describes the individual metadata for the specified object. Please find the `create` operation parameters listed here.
```

Author: Updated.

       
            - **sObjectName** : Name of the sObject which we need to create in the salesforce.
            - **fieldAndValue** : The field and value need to store in the created salesforce sObject.
        
            While invoking the API, the above two parameters values are extracted out of the user input.
    
        2. Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `create` operation into the Design panel.
    
            <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-create.png" title="Drag and drop create operation" width="800" alt="Drag and drop create operations"/>
    

>         3. To get the input values in to the API we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/) . Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design panel as shown bellow.

Reviewer: ```suggestion
        3. To get the input values into the API, we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/). Navigate into the **Palette** pane and select the graphical mediators icons listed under the **Mediators** section. Then drag and drop the `Property` mediators into the Design panel as shown below.
```

Author: Updated.

    
            <img src="../../../../assets/img/connectors/salesforce-api-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>

            The parameters available for configuring the Property mediator are as follows:
    
            > **Note**: That the properties should be add to the pallet before create the operation.
    

>         4. Add the property mediator to capture the `sObjectName` value.The sObjectName type can be used to retrieve the metadata for the Account object using the GET method, or create a new Account object using the POST method. In this example we are going to create a new Account object using the POST method.

Reviewer: ```suggestion
        4. Add the property mediator to capture the `sObjectName` value. The sObjectName type can be used to retrieve the metadata for the Account object using the GET method, or create a new Account object using the POST method. In this example we are going to create a new Account object using the POST method.
```

   
            - **name** : sObjectName
            - **expression** : json-eval($.sObject)
            - **type** : STRING
   
            <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values1.png" title="Add values to capture sObjectName value" width="800" alt="Add values to capture sObjectName value"/>
    

>         5. Add the property mediator to capture the `fieldAndValue` values. The fieldAndValue contains object fields and values that user need to store.

Reviewer: ```suggestion
        5. Add the property mediator to capture the `fieldAndValue` values. The fieldAndValue contains object fields and values that the user needs to store.
```

Author: Updated.

   
            - **name** : fieldAndValue
            - **expression** : json-eval($.fieldAndValue)
            - **type** : STRING
     
            <img src="../../../../assets/img/connectors/salesforce-api-property-mediator-values2.png" title="Add values to capture fieldAndValue value" width="800" alt="Add values to capture fieldAndValue value"/>  
    

> 3. Creating the sequence to retrive the created salesforce objects.

Reviewer: ```suggestion
3. Creating the sequence to retrieve the created Salesforce objects.
```

Author: Updated.

>    For this you need to define a new sequence called `retrive`. Please follow the steps given in 2 for create a new sequence.

Reviewer: ```suggestion
   For this you need to define a new sequence called `retrieve`. Please follow the steps given in 2 to create a new sequence.
```

Author: Updated.

    
    1. Initialize the connector.
    

>         You can use the generated tokens to initialize the connector. Please follow the steps  given in 2.1 for setting up the `init` operation to the `retrive.xml` sequence. 

Reviewer: Change the numbering. I guess this will be "2.a". Please check and update in all the sections.

Author: updated.

>             - **queryString** :  This variable contains specified SOQL query. In this sample this SOQL query executes to retrive `id` and `name` from created `Account`. If the query results are too large, the response contains the first batch of results.

Reviewer: ```suggestion
            - **queryString** :  This variable contains the specified SOQL query. In this sample, this SOQL query executes to retrieve `id` and `name` from the created `Account`. If the query results are too large, the response contains the first batch of results.
```

Author: Updated.

>         2. Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `query` operations into the Design panel.      

Reviewer: ```suggestion
        2. Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then, drag and drop the `query` operations into the Design panel.      
```

Author: Updated.

    
            <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-query.png" title="Add query operation to retrive sequnce" width="800" alt="Add query operation to retrive sequnce"/> 
    

>         3. Select the query operation and add SOQL query to the properties section shown as bellow.

Reviewer: ```suggestion
        3. Select the query operation and add the SOQL query to the properties section shown below.
```

Author: Updated.

>             - **SOQL query** : id, name from Account (Retrive list of `id`s and `name`s from the `Account` type sObject)

Reviewer: ```suggestion
            - **SOQL query** : `id` and `name` from `Account` (Retrieve list of `id`s and `name`s from the `Account` type sObject)
```

Check spelling of "retrieve" everywhere.

Author: Updated.
